src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -146,6 +147,16 @@ static void MapFocus(IEntryHandler handler, IEntry entry, object? args)
 				handler.PlatformView.Focus(request);
 		}
 
+		void OnPlatformViewAttachedToWindow(object? sender, ViewAttachedToWindowEventArgs e)
+		{
+			if (PlatformView.IsAlive() && PlatformView.Enabled)
```

Comment:
Looks good!

I'm thinking you'll be able to add visual tests for this one. If you can't get an automated test for this PR just let us know and we can add a manual one internally.

Reply:
I have added visual test for this. Please verify